packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -298,17 +306,27 @@ function FontLibraryProvider( { children } ) {
 				refreshLibrary();
 			}
 
-			if ( installationErrors.length > 0 ) {
+			if ( installationErrors.length === 1 ) {
 				throw new Error(
 					sprintf(
 						/* translators: %s: Specific error message returned from server. */
+						__( 'There was an error installing fonts. %s' ),
+						installationErrors[ 0 ]
+					)
+				);
+			} else if ( installationErrors.length > 1 ) {
+				throw new Error(
+					sprintf(
+						/* translators: %s: Specific error messages returned from server. */
 						__( 'There were some errors installing fonts. %s' ),
-						installationErrors.reduce(
-							( errorMessageCollection, error ) => {
-								return `${ errorMessageCollection } ${ error.message }`;
-							},
-							''
-						)
+						'<ul>' +
+							installationErrors.reduce(
+								( errorMessageCollection, errorMessage ) => {
+									return `${ errorMessageCollection }<li>${ errorMessage }</li>`;
```

Comment:
I noticed that if I

- upload 3 fonts
- upload the same 3 fonts again 

...I see only a single error message `A font face matching those settings already exists.` despite the fact that all three of the fonts were invalid (already existed).

It's a nit: but I wonder if we could include information in the error about the specific font that caused the error?